### ISSUE TITLE
Immunization dates after cohort end date

### ISSUE THREAD

**Issue by fsabrilb** [epiverse-trace/vaccineff#35]
Please place an "x" in all the boxes that apply
---------------------------------------------

- [ ] I have the most recent version of linelist and R
- [X] I have found a bug
- [ ] I have a [reproducible example](http://reprex.tidyverse.org/articles/reprex-dos-and-donts.html)
- [ ] I want to request a new feature

--------

Please include a brief description of the problem with a code example:

Immunization dates in a cohort's data exceed the end date of the cohort

```r
library("vaccineff")
data(cohortdata)

cohortdata$immunization <- get_immunization_date(
  data = cohortdata,
  outcome_date_col = "death_date",
  outcome_delay = 0,
  immunization_delay = 14,
  vacc_date_col = c("vaccine_date_1", "vaccine_date_2"),
  end_cohort = end_cohort,
  take_first = FALSE
)

df_ <- dplyr::count(cohortdata, immunization, name = "count_immunization_date")
dplyr::arrange(df_, desc(immunization))

```

--------

**Comment by davidsantiagoquevedo** [comment]
Solved in PR #36

**Comment by davidsantiagoquevedo** [comment]
Solved in PR #36
